shared/utils/src/test/roundvalue.unit.spec.js

```diff
@@ -9,51 +15,121 @@ tape('\n', function (test) {
 	test.end()
 })
 
-tape('roundValue tests', function (test) {
-	let value = 1.23456789e-10
-	let rounded = 1.2e-10
-	test.equal(roundValueAuto(value), rounded, `should return ${rounded}`)
+tape('roundValueAuto()', function (test) {
+	let value, rounded, formatted
+
+	value = 1.23456789e-10
+	rounded = 1.2e-10
+	test.equal(roundValueAuto(value), rounded, `Should return rounded value=${rounded} for input value=${value}`)
```

Comment:
there's anxiety whether `(value,true)` returns a different value. please add such tests

Reply:
The tests are added in https://github.com/stjude/proteinpaint/pull/2439/commits/9bb2c0ec26cef29c6a8c36b08018b55bf3246881